style/index.html

```diff
@@ -3,32 +3,112 @@
-    <button class="solve btn">Solve</button>
-    <button class="buy btn">Buy Vowel</button>
-  </section>
-
+  <main class="left-section">
+    <section class="round-container">
+      <h1 class="round-text">Round: 1</h1>
+    </section>
+    <section class="catergory-container">
+      <h2 class="catergory-text">Catergory:</h2>
+    </section>
+    <section class="board-container">
+      <div class="row1">
+        <div class="box-styles box1"></div>
+        <div class="box-styles box2"></div>
+        <div class="box-styles box3"></div>
+        <div class="box-styles box4"></div>
+        <div class="box-styles box5"></div>
+        <div class="box-styles box6"></div>
+        <div class="box-styles box7"></div>
+        <div class="box-styles box8"></div>
+        <div class="box-styles box9"></div>
+        <div class="box-styles box10"></div>
+        <div class="box-styles box11"></div>
+        <div class="box-styles box12"></div>
+        <div class="box-styles box13"></div>
+        <div class="box-styles box14"></div>
+      </div>
+      <div class="row2">
+        <div class="box-styles box15"></div>
+        <div class="box-styles box16"></div>
+        <div class="box-styles box17"></div>
+        <div class="box-styles box18"></div>
+        <div class="box-styles box19"></div>
+        <div class="box-styles box20"></div>
+        <div class="box-styles box21"></div>
+        <div class="box-styles box22"></div>
+        <div class="box-styles box23"></div>
+        <div class="box-styles box24"></div>
+        <div class="box-styles box25"></div>
+        <div class="box-styles box26"></div>
+        <div class="box-styles box27"></div>
+        <div class="box-styles box28"></div>
+      </div>
+      <div class="row3">
+        <div class="box-styles box29"></div>
+        <div class="box-styles box30"></div>
+        <div class="box-styles box31"></div>
+        <div class="box-styles box32"></div>
+        <div class="box-styles box33"></div>
+        <div class="box-styles box34"></div>
+        <div class="box-styles box35"></div>
+        <div class="box-styles box36"></div>
+        <div class="box-styles box37"></div>
+        <div class="box-styles box38"></div>
+        <div class="box-styles box39"></div>
+        <div class="box-styles box40"></div>
+        <div class="box-styles box41"></div>
+        <div class="box-styles box42"></div>
+      </div>
+      <div class="row4">
+        <div class="box-styles box43"></div>
+        <div class="box-styles box44"></div>
+        <div class="box-styles box45"></div>
+        <div class="box-styles box46"></div>
+        <div class="box-styles box47"></div>
+        <div class="box-styles box48"></div>
+        <div class="box-styles box49"></div>
+        <div class="box-styles box50"></div>
+        <div class="box-styles box51"></div>
+        <div class="box-styles box52"></div>
+        <div class="box-styles box53"></div>
```

Comment:
We only need 52 boxes for the classic WOF shape. Do you think we can use bigger boxes in the rectangle shape so that we only use something under 50 boxes?

Reply:
Nevermind! I checked out the branch and I see what you did. Looks great!